Static assert failed while building v0.13.0 with system fmt

**Describe the bug**
When compiling  v0.13.0 build fails with
```
/home/zannzetsu/open3d/src/open3d/cpp/open3d/visualization/rendering/filament/FilamentEntitiesMods.cpp:212:22:   required from here
/usr/include/fmt/core.h:1579:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1579 |       !std::is_same<decltype(arg), const unformattable&>::value,
```
 

**To Reproduce**
Steps to reproduce the behavior:
```
  mkdir build
  cd build
  find ../ -name "CMakeLists.txt" -exec sed -i 's/-Werror//g' {} \;
  cmake .. \
        -DBUILD_SHARED_LIBS=ON \
        -DCMAKE_BUILD_TYPE=Release \
        -DUSE_SYSTEM_FLANN=ON \
        -DUSE_SYSTEM_FMT=ON \
        -DUSE_SYSTEM_GLEW=ON \
        -DUSE_SYSTEM_GLFW=ON \
        -DUSE_SYSTEM_GOOGLETEST=ON \
        -DUSE_SYSTEM_JPEG=ON \
        -DUSE_SYSTEM_LIBLZF=ON \
        -DUSE_SYSTEM_PNG=ON \
        -DUSE_SYSTEM_PYBIND11=ON
  make -j4
```

**Expected behavior**
The build should succeed despite using system FMT.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**

- Operating system: Archlinux x86_64
- Python version: Python 3.9.6
- Open3D version: v0.13.0
- Is this remote workstation?: no
- How did you install Open3D?: I try to maintain an AUR package, please consider this a from source build
- Compiler version (if built from source): gcc (GCC) 11.1.0, clang version 12.0.1

**Additional context**
System fmt 8.0.0-2

I just tried, it failed with a different error message.
```
cmake .. \
        -DBUILD_SHARED_LIBS=ON \
        -DCMAKE_BUILD_TYPE=Release \
        -DUSE_SYSTEM_FLANN=ON \
        -DUSE_SYSTEM_FMT=ON \
        -DUSE_SYSTEM_GLEW=ON \
        -DUSE_SYSTEM_GLFW=ON \
        -DUSE_SYSTEM_GOOGLETEST=ON \
        -DUSE_SYSTEM_JPEG=ON \
        -DUSE_SYSTEM_LIBLZF=ON \
        -DUSE_SYSTEM_PNG=ON \
        -DUSE_SYSTEM_PYBIND11=ON
-- CMAKE_BUILD_TYPE is set to Release.
-- Downloading 3rdparty dependencies to /home/zannzetsu/opend3d/Open3D/3rdparty_downloads
-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Open3D 0.13.0+c86717bb7
-- Found Python3: /usr/bin/python3.9 (found suitable version "3.9.6", required range is "3.6...<3.10") found components: Interpreter Development Development.Module Development.Embed 
-- Performing Test FLAG_fstack_protector_strong
-- Performing Test FLAG_fstack_protector_strong - Success
-- Performing Test FLAG_Wformat
-- Performing Test FLAG_Wformat - Success
-- Performing Test FLAG_Wformat_security
-- Performing Test FLAG_Wformat_security - Success
-- Performing Test FLAG_Wl__z_noexecstack
-- Performing Test FLAG_Wl__z_noexecstack - Success
-- Performing Test FLAG_Wl__z_relro__z_now
-- Performing Test FLAG_Wl__z_relro__z_now - Success
-- Performing Test FLAG_pie
-- Performing Test FLAG_pie - Success
-- Performing Test FLAG____CONFIG_Release__LINKER___strip_debug_
-- Performing Test FLAG____CONFIG_Release__LINKER___strip_debug_ - Success
-- Using security hardening compiler flags: -fstack-protector-strong;-Wformat;-Wformat-security and
linker flags: -Wl,-z,noexecstack;-Wl,-z,relro,-z,now;-pie;$<$<CONFIG:Release>:LINKER:--strip-debug>
-- Performing Test HAS_GLIBCXX_USE_CXX11_ABI
-- Performing Test HAS_GLIBCXX_USE_CXX11_ABI - Success
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Using installed third-party library 3rdparty_threads 
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Using installed third-party library 3rdparty_openmp 
-- Building with OpenMP
-- Using installed third-party library 3rdparty_x11 
-- Found GLEW: /usr/include (found version "2.2.0") 
-- Using installed third-party library 3rdparty_glew 2.2.0
-- Using installed third-party library 3rdparty_glfw3 3.3.4
-- Found JPEG: /usr/lib/libjpeg.so (found version "80") 
-- Using installed third-party library 3rdparty_jpeg 80
CMake Warning at 3rdparty/find_dependencies.cmake:253 (find_package):
  By not providing "Findliblzf.cmake" in CMAKE_MODULE_PATH this project has                                                                                                       
  asked CMake to find a package configuration file provided by "liblzf", but                                                                                                      
  CMake did not find one.                                                                                                                                                         

  Could not find a package configuration file provided by "liblzf" with any                                                                                                       
  of the following names:                                                                                                                                                         

    liblzfConfig.cmake                                                                                                                                                            
    liblzf-config.cmake                                                                                                                                                           

  Add the installation prefix of "liblzf" to CMAKE_PREFIX_PATH or set                                                                                                             
  "liblzf_DIR" to a directory containing one of the above files.  If "liblzf"                                                                                                     
  provides a separate development package or SDK, be sure it has been                                                                                                             
  installed.                                                                                                                                                                      
Call Stack (most recent call first):                                                                                                                                              
  3rdparty/find_dependencies.cmake:674 (open3d_find_package_3rdparty_library)                                                                                                     
  CMakeLists.txt:646 (include)                                                                                                                                                    


-- Unable to find installed third-party library 3rdparty_lzf
-- Found ZLIB: /usr/lib/libz.so (found version "1.2.11") 
-- Found PNG: /usr/lib/libpng.so (found version "1.6.37") 
-- Using installed third-party library 3rdparty_libpng 
-- Using installed third-party library 3rdparty_fmt 8.0.0
-- Performing Test HAS_FLTO
-- Performing Test HAS_FLTO - Success
-- Found pybind11: /usr/include (found version "2.6.2" )
-- Found OpenGL: /usr/lib/libOpenGL.so   
-- Using installed third-party library 3rdparty_opengl 
-- Using prebuilt third-party library Filament
-- GLIBC version 2.33 found: Using Open3D Filament binary.
-- Filament is located at /home/zannzetsu/opend3d/Open3D/build/filament/src/ext_filament
-- CLANG_LIBDIR: /usr/lib
-- CPP_LIBRARY: /usr/lib/libc++.so
-- CPPABI_LIBRARY: /usr/lib/libc++abi.so
-- Using MKL to support BLAS and LAPACK functionalities.
-- Building third-party library faiss from source
-- IPP-ICV 2020.0.0 Gold available. Building interface wrappers IPP-IW.
-- Copying /home/zannzetsu/opend3d/Open3D/cpp/open3d/visualization/webrtc_server/html to /home/zannzetsu/opend3d/Open3D/build/bin/resources.
-- PYPI_PACKAGE_NAME: open3d
-- ================================================================================
-- Open3D 0.13.0 Configuration Summary
-- ================================================================================
-- Enabled Features:
--   OpenMP .................................. ON
--   Headless Rendering ...................... OFF
--   Azure Kinect Support .................... OFF
--   Intel RealSense Support ................. OFF
--   CUDA Support ............................ OFF
--   Build GUI ............................... ON
--   Build WebRTC visualizer ................. ON
--   Build Shared Library .................... ON
--   Build Unit Tests ........................ OFF
--   Build Examples .......................... ON
--   Build Python Module ..................... ON
--   Build Jupyter Extension ................. OFF
--   Build TensorFlow Ops .................... OFF
--   Build PyTorch Ops ....................... OFF
--   Build Benchmarks ........................ OFF
--   Bundle Open3D-ML ........................ OFF
--   Build RPC interface ..................... OFF
--   Force GLIBCXX_USE_CXX11_ABI= ............ 0
-- ================================================================================
-- Third-Party Dependencies:
--   EIGEN3 .................................. no
--   FAISS ................................... no
--   FILAMENT ................................ no
--   FMT ..................................... no
--   GLEW .................................... no
--   GLFW .................................... no
--   GOOGLETEST .............................. no
--   IMGUI ................................... no
--   IPPICV .................................. no
--   JPEG .................................... no
--   JSONCPP ................................. no
--   LIBLZF .................................. no
--   OPENGL .................................. no
--   PNG ..................................... no
--   PYBIND11 ................................ no
--   QHULL ................................... no
--   LIBREALSENSE ............................ no
--   TINYFILEDIALOGS ......................... no
--   TINYGLTF ................................ no
--   TINYOBJLOADER ........................... no
--   WEBRTC .................................. no
-- ================================================================================
-- Configuring done
CMake Error: install(EXPORT "Open3DTargets" ...) includes target "Open3D" which requires target "3rdparty_fmt" that is not in any export set.
CMake Error in CMakeLists.txt:
  export called with target "Open3D" which requires target "3rdparty_fmt"                                                                                                         
  that is not in any export set.                                                                                                                                                  


-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    USE_SYSTEM_FLANN


CMake Generate step failed.  Build files cannot be regenerated correctly
```

I see that DUSE_SYSTEM_FLANN was removed so I won't use it anymore. liblzf is installed in version `3.6-2`.

--- a/cpp/open3d/utility/Logging.h
+++ b/cpp/open3d/utility/Logging.h
@@ -42,6 +42,7 @@
 #endif
 #endif
 #include <fmt/format.h>
+#include <fmt/ostream.h>
 #include <fmt/printf.h>
 #include <fmt/ranges.h>
 

